go/rows.go

```diff
@@ -206,18 +207,15 @@ func (r *Rows) openResults() error {
 		if err != nil {
 			return
 		}
-		err = scratchFile.Close()
```

Comment:
I think `scratchFile` and `bufferedFile` are a writer and reader for the same underlying file; is it a problem if we don't `Close` before we start reading?

Reply:
I think it should be safe (example here: https://goplay.tools/snippet/BMTZVuBErrX). I can rip that out though. I was just concerned `scratchFile` may remain open if the initial call to `DownloadWithContext` errs

Reply:
Good point.

My only concern was that something might not be flushed, but after re-reading the docs and playing with your example, I'm convinced this is safe.

🚢 🇮🇹